src/unlinkcommand.js

```diff
@@ -14,6 +14,27 @@ import findLinkRange from './findlinkrange.js';
  */
 export default class UnlinkCommand extends Command {
 	/**
+	 * @see core.command.Command
+	 * @param {core.editor.Editor} editor
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		/**
+		 * Flag indicating whether command is active. For collapsed selection it means that typed characters will have
+		 * the command's attribute set. For range selection it means that all nodes inside have the attribute applied.
+		 *
+		 * @observable
+		 * @member {Boolean} core.command.ToggleAttributeCommand#value
+		 */
+		this.set( 'hasValue', undefined );
```

Comment:
Is `hasValue` some kind of standard in the codebase? Because to me it's not very intuitive. It should be more like `isOn` (might be confusing because in v4 we used `isOff`) or `isEnabled` or sth. else.